.chloggen/instana-cafile-config-option.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: instanaexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Added a new configuration option ca_file that accepts a CA certificate that allows the exporter to make HTTPS requests to a self signed backend 
```

Comment:
Would this read better as:

Added a new configuration option ca_file that accepts a CA certificate that allows the exporter --> Added a new configuration option ca_file which takes the certificate authority file for an Instana backend with a self signed certificate